package.json

```diff
@@ -2,8 +2,9 @@
   "name": "react-client-store",
   "version": "1.0.0-rc.1",
   "description": "",
-  "source": "src/index.js",
+  "source": "src/index.tsx",
   "main": "dist/index.js",
+  "types": "dist/index.d.ts",
```

Comment:
i think this should be `dist/types.d.ts`. No matter what we specify here, Parcel will somehow output types.d.ts

Reply:
Tested on Windows 10 and Ubuntu on WSL2, the typing output is based on what we specify in `types`, so in this case it will generate `dist/index.d.ts` as expected

Generated info:
![image](https://user-images.githubusercontent.com/50788123/236750509-f6e12a52-9e40-4954-ba21-d57b1e4f0acb.png)

Detail output:
![image](https://user-images.githubusercontent.com/50788123/236750729-5edc2f4a-4cd3-49ea-b21b-c8c61e5304fc.png)

References:
- https://parceljs.org/languages/typescript/#generating-typings

Reply:
You're right. Turned out it was actually a parcel caching issue on my end.